src/scanpy/plotting/_dotplot.py

```diff
@@ -303,18 +301,18 @@ def __init__(
     def style(
         self,
         *,
-        cmap: str = DEFAULT_COLORMAP,
-        color_on: Literal["dot", "square"] | None = DEFAULT_COLOR_ON,
-        dot_max: float | None = DEFAULT_DOT_MAX,
-        dot_min: float | None = DEFAULT_DOT_MIN,
-        smallest_dot: float | None = DEFAULT_SMALLEST_DOT,
-        largest_dot: float | None = DEFAULT_LARGEST_DOT,
-        dot_edge_color: ColorLike | None = DEFAULT_DOT_EDGECOLOR,
-        dot_edge_lw: float | None = DEFAULT_DOT_EDGELW,
-        size_exponent: float | None = DEFAULT_SIZE_EXPONENT,
-        grid: float | None = False,
-        x_padding: float | None = DEFAULT_PLOT_X_PADDING,
-        y_padding: float | None = DEFAULT_PLOT_Y_PADDING,
+        cmap: str | None | Empty = _empty,
```

Comment:
Why can this be `None` now?

Reply:
and others

Reply:
`self.cmap` was always allowed to be None, the type annotations for `style` were just wrong.